src/core/client/stream/tabs/Comments/Stream/StreamContainer.tsx

```diff
@@ -147,57 +147,68 @@ export const StreamContainer: FunctionComponent<Props> = props => {
           />
         )}
         <HorizontalGutter spacing={4} className={styles.tabBarContainer}>
-          <SortMenu
-            className={styles.sortMenu}
-            orderBy={local.commentsOrderBy}
-            onChange={onChangeOrder}
-            reactionSortLabel={props.settings.reaction.sortLabel}
-          />
-          <TabBar
-            variant="secondary"
-            activeTab={local.commentsTab}
-            onTabClick={onChangeTab}
-            className={CLASSES.tabBarComments.$root}
-          >
-            {featuredCommentsCount > 0 && (
-              <TabWithFeaturedTooltip tabID="FEATURED_COMMENTS">
-                <Flex spacing={1} alignItems="center">
-                  <Localized id="comments-featuredTab">
-                    <span>Featured</span>
+          <Flex>
+            <TabBar
+              variant="secondary"
+              activeTab={local.commentsTab}
+              onTabClick={onChangeTab}
+              className={cn(styles.tabBar, CLASSES.tabBarComments.$root)}
+            >
+              {featuredCommentsCount > 0 && (
+                <TabWithFeaturedTooltip tabID="FEATURED_COMMENTS">
+                  <Flex spacing={1} alignItems="center">
+                    <Localized id="comments-featuredTab">
+                      <span className={styles.featuredTabText}>Featured</span>
+                    </Localized>
+                    <Counter
+                      data-testid="comments-featuredCount"
+                      size="sm"
+                      color={
+                        local.commentsTab === "FEATURED_COMMENTS"
+                          ? "primary"
+                          : "grey"
+                      }
+                    >
+                      {featuredCommentsCount}
+                    </Counter>
+                  </Flex>
+                </TabWithFeaturedTooltip>
+              )}
+              <Tab
+                tabID="ALL_COMMENTS"
+                classes={{
+                  root: styles.allCommentsTabContainer,
+                }}
+                className={cn(
+                  styles.allCommentsTab,
+                  CLASSES.tabBarComments.allComments
+                )}
+              >
+                <Flex alignItems="center" spacing={1}>
+                  <Localized id="comments-allCommentsTab">
+                    <span className={styles.allCommentsTabText}>
+                      All Comments
+                    </span>
                   </Localized>
                   <Counter
-                    data-testid="comments-featuredCount"
                     size="sm"
                     color={
-                      local.commentsTab === "FEATURED_COMMENTS"
-                        ? "primary"
-                        : "grey"
+                      local.commentsTab === "ALL_COMMENTS" ? "primary" : "grey"
                     }
                   >
-                    {featuredCommentsCount}
+                    {allCommentsCount}
```

Comment:
git is having trouble with this diff, fewer lines are changed than it is showing